### DISCUSSION TITLE
v0.12.0-beta.6

### DISCUSSION THREAD

**Discussion by alexjustesen** [alexjustesen/speedtest-tracker#775] [Releases]
## What's Changed
* [Feature] User role by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/762
* Laravel 10.23.0 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/764
* Laravel 10.23.1 Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/768
* Laravel 10 Mid-Shift by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/771
* added back collapsible sidebar by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/772
* [Bug] Settings page authorization by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/773
* Release v0.12.0.beta.6 by @alexjustesen in https://github.com/alexjustesen/speedtest-tracker/pull/774

**Full Changelog**: https://github.com/alexjustesen/speedtest-tracker/compare/v0.12.0-beta.5...v0.12.0-beta.6

<hr /><em>This discussion was created from the release <a href='https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.12.0-beta.6'>v0.12.0-beta.6</a>.</em>

**Comment by jiriteach** [comment]
@alexjustesen - Didnt want to open a bug without confirming, but changing the site name under General Settings does nothing? If I have APP_NAME set in .env - it picks that up else it defaults to Speedtest Traacker and setting site name doesnt do anything. Any ideas?

**Reply by alexjustesen** [reply]
This sounds correct, `APP_NAME` should take priority over the settings option. Now that I'm moving to an environment variable based configuration this setting will get removed in a future version.

**Reply by jiriteach** [reply]
Thanks. If I remove APP_NAME from .env, how should we control setting the name as the Site Name in settings doesnt do anything.

**Reply by alexjustesen** [reply]
I'd stick with `APP_NAME` as an environment variable, I'm going to remove the setting in a future release.
